README.md

```diff
@@ -118,26 +120,27 @@ sap_hana_preconfigure_fail_if_reboot_required
 ```
 
 ### Switch to tuned profile sap-hana
-If you do not want to have the role switch to tuned profile sap-hana, set the following variable to `no`. Default is `yes`.
+If you do not want the role to switch to tuned profile sap-hana, set the following variable to `no`. Default is `yes`. In case of `yes`, variable `sap_hana_preconfigure_use_tuned_where_possible` (see below) should be set to `yes` as well.
 ```yaml
 sap_hana_preconfigure_switch_to_tuned_profile_sap_hana
 ```
 
 ### Use tuned profile sap-hana where possible
-If you do not want to use the tuned profile sap-hana for configuring kernel parameters (where possible), and have the role configure them instead, set the following variable to `no`. Default is `yes`.
+If you do not want to use the tuned profile sap-hana for configuring kernel parameters (where possible), and have the role configure them by changing the kernel command line instead, set the following variable to `no`. Default is `yes`. In case of `yes`, variable `sap_hana_preconfigure_switch_to_tuned_profile_sap_hana` (see above) should be set to `yes` as well.
+Note: If this variable is set to `yes`, the role will not modify GRUB_CMDLINE_LINUX in /etc/default/grub, no matter how `sap_hana_preconfigure_modify_grub_cmdline_linux` (see below) is set.
```

Comment:
@berndfinger not suggesting you do this now, but possibly for a future release - did you investigate using the tuned profile to configure the bootloader cmdline?

Reply:
Yes, I found that the three kernel boot cmdline parameters which are configured by this role are all covered by tuned-profiles-sap-hana. Here's the list:
CMDLINE: transparent_hugepage=never
tuned-profiles-sap-hana: transparent_hugepages=never
See https://github.com/berndfinger/sap-hana-preconfigure/blob/006a1c907953382860486528cdff13c344632309/tasks/RedHat/generic/disable-thp.yml#L6

CMDLINE: processor.max_cstate=1 intel_idle.max_cstate=1
tuned-profiles-sap-hana: force_latency=70 or force_latency=cstate.id:3|70
See https://github.com/berndfinger/sap-hana-preconfigure/blob/006a1c907953382860486528cdff13c344632309/tasks/RedHat/generic/configure-c-states-for-lower-latency.yml#L6